CHANGELOG.md

```diff
@@ -63,6 +63,7 @@ For older changes see the [archived Singularity change log](https://github.com/a
 - The `instance start` command now accepts an optional `--app <name>` argument which
   invokes start script within the `%appstart <name>` section in the definition file.
   The `instance stop` command still only requires the instance name.
+- Templating Support for Definition File.
```

Comment:
Please expand this to mention the new `--build-arg` and `--build-arg-file` options for replacing variables in definition files that are surrounded by a matching pair of double curly brackets `{{ ... }}`.

Reply:
thanks! resolved in the latest commit.
https://github.com/apptainer/apptainer/pull/1344/commits/1842bc374a7db1c7a72fb8fa11d811d7de4e2ff4